internal/commands/sbomtest/flags.go

```diff
@@ -0,0 +1,17 @@
+package sbomtest
+
+import "github.com/spf13/pflag"
+
+const (
+	SBOMFlagExperimental = "experimental"
+	SBOMFlagFile         = "file"
```

Comment:
can we add to `flags.go`? The exported function should probably be renamed from `GetSBOMFlagSet()` to `GetSBOMTestFlagSet()` if being moved to the `flags` package.

Reply:
oh, trigger happy merging 😄 i'll create a PR to suggest a refactor

Reply:
I thought keeping the flags by the command would be helpful as they are specific to the command.

I am putting together another PR where the existing command is moved into its own folder and was considering moving the existing `flags.go` along with the command. What are you thoughts?

Reply:
ah, gotcha. thoughts are that we are using flags that are already being defined for the existing command and historically this duplication can result in effects of UI inconsistencies. in my experience it's not going to have them a bit further away from the command, you usually just set them up once and forget that they even exist.